.github/workflows/playwright.yml

```diff
@@ -43,3 +53,6 @@ jobs:
           name: playwright-reports
           path: playground/*/playwright-report/
```

Comment:
now that we have the site matrix, we should only upload the report for that site

Reply:
also, the names should be different for each run so they don't write over each other